Nos/Views/Home/HomeFeedView.swift

```diff
@@ -188,57 +188,14 @@ struct HomeFeedView: View {
     }
 }
 
-struct ContentView_Previews: PreviewProvider {
+#Preview {
```

Comment:
💯 glad we're on the same page about replacing `PreviewProvider` with `#Preview`.